Firmware/Pluma-Adafruit-BNO055/Pluma-Adafruit-BNO055.ino

```diff
@@ -0,0 +1,195 @@
+    digitalWrite(LED_pin,my_val);
+
+
+
+      //could add VECTOR_ACCELEROMETER, VECTOR_MAGNETOMETER,VECTOR_GRAVITY...
+      sensors_event_t orientationData , angVelocityData , linearAccelData, magnetometerData, accelerometerData, gravityData;
+      bno.getEvent(&orientationData, Adafruit_BNO055::VECTOR_EULER);
+      bno.getEvent(&angVelocityData, Adafruit_BNO055::VECTOR_GYROSCOPE);
+      bno.getEvent(&linearAccelData, Adafruit_BNO055::VECTOR_LINEARACCEL);
+      bno.getEvent(&magnetometerData, Adafruit_BNO055::VECTOR_MAGNETOMETER);
+      bno.getEvent(&accelerometerData, Adafruit_BNO055::VECTOR_ACCELEROMETER);
+      bno.getEvent(&gravityData, Adafruit_BNO055::VECTOR_GRAVITY);
+
+      printEvent(&orientationData);
+      printEvent(&angVelocityData);
+      printEvent(&linearAccelData);
+      printEvent(&magnetometerData);
+      printEvent(&accelerometerData);
+      printEvent(&gravityData);
+
+
+
+      uint8_t system, gyro, accel, mag = 0;
+      bno.getCalibration(&system, &gyro, &accel, &mag);
+      //Serial.println();
+      Serial.print("Calibration:Sys=");
+      Serial.print(system);
+      Serial.print(",Gyro=");
+      Serial.print(gyro);
+      Serial.print(",Accel=");
+      Serial.print(accel);
+      Serial.print(",Mag=");
+      Serial.print(mag);
+
+
+      int8_t boardTemp = bno.getTemp();
+      Serial.print(F(";Temp:value="));
+      Serial.print(boardTemp);
+      Serial.println(";");
+
+      delay(BNO055_SAMPLERATE_DELAY_MS);
+  }
+
+
+
+  
+}
+
+
+
+// ----------------------------------------------------------------------------------------
+void printEvent(sensors_event_t* event) {
+  double x = -1000000, y = -1000000 , z = -1000000; //dumb values, easy to spot problem
+  if (event->type == SENSOR_TYPE_ACCELEROMETER) {
+    Serial.print("Accl:");
+    x = event->acceleration.x;
+    y = event->acceleration.y;
+    z = event->acceleration.z;
+  }
+  else if (event->type == SENSOR_TYPE_ORIENTATION) {
+    Serial.print("Orient:");
+    x = event->orientation.x;
+    y = event->orientation.y;
+    z = event->orientation.z;
+  }
+  else if (event->type == SENSOR_TYPE_MAGNETIC_FIELD) {
+    Serial.print("Mag:");
+    x = event->magnetic.x;
+    y = event->magnetic.y;
+    z = event->magnetic.z;
+  }
+  else if (event->type == SENSOR_TYPE_GYROSCOPE) {
+    Serial.print("Gyro1:");
+    x = event->gyro.x;
+    y = event->gyro.y;
+    z = event->gyro.z;
+  }
+  else if (event->type == SENSOR_TYPE_ROTATION_VECTOR) {
+    Serial.print("Rot:");
+    x = event->gyro.x;
+    y = event->gyro.y;
+    z = event->gyro.z;
+  }
+  else if (event->type == SENSOR_TYPE_LINEAR_ACCELERATION) {
+    Serial.print("Linear:");
+    x = event->acceleration.x;
+    y = event->acceleration.y;
+    z = event->acceleration.z;
+  }
+  else if (event->type == SENSOR_TYPE_GRAVITY) {
+    Serial.print("Gravity:");
+    x = event->acceleration.x;
+    y = event->acceleration.y;
+    z = event->acceleration.z;
+  }
+  else {
+    Serial.print("Unk:");
```

Comment:
Same as above, maybe there would be a way to add a prefix to the log and error messages, e.g. `Log:`, `Error:` so we can parse them on the high-level interface and either ignore or throw?

Reply:
3. if same as above,  "_Should we keep (...) warning messages? high-level interface might have trouble parsing these messages_" , as I stated to remove that warnings can/should be removed.
Then it would be consistent to remove this warning "Unk:" too.

Reply:
Promoted the `Unk;` to `Error:` because it keeps the same number of elements, and in future messages the hardware might recover. The values sent in this case are `x = -1000000, y = -1000000, z = -1000000;` 
According to Adafruit sample code `// dumb values, easy to spot problem`

Reply:
Fix in 2248ad1fbffe843183a1e15099b95815f06d5661